pagopa/io-app
# chore: [IA-143] Updated onboarding ShareDataComponent + back button

## Short description
This PR is going to implement a small layout update to the `ShareDataComponent` and add a back button to the `OnboardingShareDataScreen` used to abort the onboarding.

The screen title in the onboarding process has been updated to `Configura IO` (For the english version I used `Configure IO`, is it alright?)

|Before|After|
|------|-----|
|![Simulator Screen Shot - iPhone 12 - 2022-02-09 at 12 29 52](https://user-images.githubusercontent.com/5963683/153197103-87daf868-7af0-4d13-b66f-afba606ac26e.png) | ![Simulator Screen Shot - iPhone 12 - 2022-02-09 at 12 25 55](https://user-images.githubusercontent.com/5963683/153197123-583071d3-8e86-4819-b890-86b153b83aa4.png)  |
| | ![Simulator Screen Shot - iPhone 12 - 2022-02-09 at 12 25 58](https://user-images.githubusercontent.com/5963683/153197560-b5ab5b68-e83d-4952-ad9e-f05518c09607.png) |


## List of changes proposed in this pull request
- The CTA for each paragraph is now inline with the text
- Added a `.` (full-stop) at the end of each paragraph
- Implemented a back button to abort the onboarding process during the share data check
- Updated the onboarding screen title from `Privacy` to `Configura IO` (`Configure IO` in english)

## How to test
In an newly installed app (or clearing all the AsyncStorage data), execute the sign in using the preferred method and this screen should be displayed. After the onboarding the same screen is accessible from the preferences.
